sfs/array.py

```diff
@@ -518,5 +621,25 @@ def _linear_helper(ycoordinates, center, orientation):
 
 
 def concatenate(*arrays):
-    """Concatenate `ArrayData` objects."""
+    """Concatenate `ArrayData` objects.
+
+    Returns
+    -------
+    `ArrayData`
+        Positions, orientations and weights of the concatenated secondary sources.
```

Comment:
sfs/array.py:629:80: E501 line too long (82 > 79 characters)

In this case, the limit should actually be 72, see https://www.python.org/dev/peps/pep-0008/#maximum-line-length.